### DISCUSSION TITLE
Why isnt the camera for the printer in the camera domain in home assistant?

### DISCUSSION THREAD

**Discussion by trwy7** [greghesp/ha-bambulab#569] [Q&A] [answered]
Not having the printer set as a camera causes issues where some other integrations wont work with it. It also shows an item in the home assistant logs whenever a snapshot of the camera is taken. Using the default camera domain will allow for the integration to only stream the camera when a user is requesting the camera.

**Comment by nzrutman** [comment, marked as answer]
https://github.com/greghesp/ha-bambulab/issues/531
